packages/sitecore-jss-react/src/components/FEaaSComponent.tsx

```diff
@@ -84,9 +84,10 @@ export const FEaaSComponent = ({ params, fields }: FEaaSComponentProps): JSX.Ele
 
   return (
     <>
-      <feaas-stylesheet library={reqProps.library} cdn={reqProps.cdn} >
-        <link rel="stylesheet" href={`${reqProps.cdn}/styles/${reqProps.library}/published.css`} />
-      </feaas-stylesheet>
+      <link rel="preload" as="style" href={`${reqProps.cdn}/styles/${reqProps.library}/published.css`} />
+      <link rel="preload" as="fetch" href={`${reqProps.cdn}/components/${reqProps.library}/${reqProps.component}/${reqProps.version}/${reqProps.revision}.html`} />
```

Comment:
Should this be as `document`?
```suggestion
      <link rel="preload" as="document" href={`${reqProps.cdn}/components/${reqProps.library}/${reqProps.component}/${reqProps.version}/${reqProps.revision}.html`} />
```

Reply:
These type match the content type that is being requested with fetch() call. So there's actually a requirement that this type needs to be the same on both ends (in our SDK and in here). Otherwise it can cause duplicate request. So maybe document could work, but since we do fetch() call to fetch the content i tested that it does duplicate request.

Maybe i could make it work with document too, but that needs some extra testing. Maybe i can do it on monday

Reply:
Apparently as="document" doesnt even work. Still open bug in firefox, and i couldn't make it in Chrome. It's supposed to be used for iframes also, so it seems not a good match semantically

Reply:
@Inviz Got it, fetch it is then!